sdk/service/service.go

```diff
@@ -70,12 +70,11 @@ func (s *Service) Create(srv *service.Service) (*service.Service, error) {
 		return nil, errors.New("service is already deployed")
 	}
 
-	// build service's Docker image and apply to service.
-	imageHash, err := s.container.Build(path)
+	// build service's Docker image.
+	_, err = s.container.Build(path)
```

Comment:
We are still building the image for test purpose but not saving the image hash here. It will be saved in instance.create.

Reply:
still valid to build to test the validity of the service